packages/salesforcedx-vscode-core/src/constants.ts

```diff
@@ -25,6 +25,12 @@ export const TELEMETRY_OPT_OUT_LINK =
   'https://developer.salesforce.com/tools/vscode/en/faq/telemetry';
 export const TELEMETRY_METADATA_COUNT = 'metadataCount';
 
+export const CONFIG_SET_EXECUTOR = 'force_config_set_org_text';
+export const CONFIG_SET_NAME = 'force_config_set_title';
+export const TABLE_NAME_COL = 'table_header_name';
+export const TABLE_VAL_COL = 'table_header_value';
+export const TABLE_SUCCESS_COL = 'table_header_success';
```

Comment:
So close to alpha order...wanna swap the last two?

Reply:
Thanks for pointing this one out.

Reply:
Checked it by swapping the order! It throws a linting error.